.github/workflows/super-linter.yaml

```
@@ -0,0 +1,38 @@
+---
+name: Lint Codebase
+
+on:
+  push:
+    branches: [master, release-*]
+  pull_request:
+    branches: [master, release-*]
+
+permissions:
+  contents: read
+  statuses: write
+
+jobs:
+  super-linter:
+    name: Super-Linter
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v3
+        with:
+          # Full git history is needed to get a proper list of changed files within super-linter
+          fetch-depth: 0
+
+      - name: Lint Code Base
+        uses: github/super-linter/slim@v4.10.0
+        env:
```

```suggestion
        env:
          LOG_LEVEL: NOTICE
```
This seems to quiet down the very verbose (pun intended) printing that I'm not sure is helpful.